modules/rhythmoneBidAdapter.js

```diff
@@ -1,286 +1,201 @@
-    const slotMap = {};
-    const bidParams = getBidParameters(params.bids);
+    return [{
+      method: 'GET',
+      url: getRMPUrl()
+    }];
+  };
 
-    debug = (bidParams !== null && bidParams.debug === true);
+  this.interpretResponse = function (serverResponse) {
+    let responses = serverResponse.body || [];
+    let bids = [];
+    let i = 0;
 
-    if (bidParams === null) {
-      noBids(params);
-      return;
+    if (responses.seatbid) {
+      let temp = [];
+      for (i = 0; i < responses.seatbid.length; i++) {
+        for (let j = 0; j < responses.seatbid[i].bid.length; j++) {
+          temp.push(responses.seatbid[i].bid[j]);
+        }
+      }
+      responses = temp;
     }
 
-    for (var i = 0; i < params.bids.length; i++) { slotMap[params.bids[i].placementCode] = params.bids[i]; }
-
-    loadStart = (new Date()).getTime();
-    load(bidParams, getRMPURL(bidParams, params.bids), function(code, msg, txt) {
-      // send quality control beacon here
-      sendAuditBeacon(bidParams.placementId);
-
-      logToConsole('response text: ' + txt);
-
-      if (code !== -1) {
-        try {
-          const result = JSON.parse(txt);
-          const registerBid = function registerBid(bid) {
-            slotMap[bid.impid].success = 1;
-
-            const pbResponse = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
-            const placementCode = slotMap[bid.impid].placementCode;
-
-            placementCodes[placementCode] = false;
-
-            pbResponse.bidderCode = bidderCode;
-            pbResponse.cpm = parseFloat(bid.price);
-            pbResponse.width = bid.w;
-            pbResponse.height = bid.h;
-
-            if ((/video/i).test(slotMap[bid.impid].mediaType)) {
-              pbResponse.mediaType = 'video';
-              pbResponse.vastUrl = bid.nurl;
-              pbResponse.descriptionUrl = bid.nurl;
-            } else { pbResponse.ad = bid.adm; }
-
-            logToConsole('registering bid ' + placementCode + ' ' + JSON.stringify(pbResponse));
-
-            bidManager.addBidResponse(placementCode, pbResponse);
-          };
-
-          for (i = 0; result.seatbid && i < result.seatbid.length; i++) {
-            for (var j = 0; result.seatbid[i].bid && j < result.seatbid[i].bid.length; j++) {
-              registerBid(result.seatbid[i].bid[j]);
-            }
-          }
-        } catch (ex) {}
+    for (i = 0; i < responses.length; i++) {
+      let bid = responses[i];
+      let bidRequest = slotsToBids[bid.impid];
+      let bidResponse = {
+        requestId: bidRequest.bidId,
+        bidderCode: that.code,
+        cpm: parseFloat(bid.price),
+        width: bid.w,
+        height: bid.h,
+        creativeId: bid.crid,
+        currency: 'USD',
+        netRevenue: true,
+        ttl: 1000
+      };
+
+      if (bidRequest.mediaTypes && bidRequest.mediaTypes.video) {
+        bidResponse.vastUrl = bid.nurl;
+        bidResponse.descriptionUrl = bid.nurl;
```

Comment:
`descriptionUrl` isn't required to be set anymore, `vastUrl` alone is fine

Reply:
Thanks, will remove that line